### DISCUSSION TITLE
command: setspawn

### DISCUSSION THREAD

**Discussion by FBDKPlus** [Zrips/CMI#7094] [Q&A]
I want to set the main city through instructions. Although your CMI plug-in has coordinates and world setting instructions, they have not taken effect!

i use command：/setspawn true -g:sorth -rng:0 -w:YCraft_ 20V20 loc:YCraft_ 20V20; 0 l74;- 155;- 0.45004272;- two point seven zero zero zero nine three three

He told me the format was wrong

**Comment by mrfloris** [comment]
<img width="574" alt="Screenshot 2022-08-20 at 13 27 51" src="https://user-images.githubusercontent.com/28841349/185743851-77e02e6c-6078-4d3f-a65c-f9bb30d1ce06.png">
Here's the syntax.

You can just go to the world you want to set the spawn point, stand in that spot exactly how you want and type /cmi setspawn, and you're done. You can do this per world. Note that if you use multiverse-core or another world manager, you need to use /mvsetspawn as well. 

If you want respawn points, you can control respawn priority in those plugins, as well as cmi's config.yml
